_i18n/en.yml

```diff
@@ -802,29 +856,22 @@ help_pages:
       For additional Trusted Traveler related questions such as these, please contact CBP directly: [https://help.cbp.gov/app/ask](https://help.cbp.gov/app/ask){:target="_blank"}.
     family-account: |-
       The [Trusted Traveler Programs site](https://ttp.cbp.dhs.gov/){:target="_blank"} supports only one login.gov account per individual application. Each family member will need a unique email address to create a login.gov account. We completely understand you may prefer to share an email address with your family members, or do not wish to set up an email account for your children.
-
-      Some email providers may have options to create alias addresses (or, forwarding addresses). If you are a Gmail user, you can use this trick to get multiple email aliases to point to the same inbox:
-
+ 
+      Some email providers may have options to create alias addresses (or, forwarding addresses). If you are a Gmail user, you have multiple email aliases point to the same inbox:
+      
```

Comment:
If you are a Gmail user you **can** have multiple email aliases...

Reply:
@clantosswett ☝️